packages/datadog-plugin-mariadb/test/index.spec.js

```diff
@@ -57,7 +57,8 @@ describe('Plugin', () => {
 
         withPeerService(
           () => tracer,
-          (done) => connection.query('SELECT 1', (_) => { done() }),
+          'mariadb',
+          done => connection.query('SELECT 1', (err) => { err && done(err) }),
```

Comment:
This used to always call done but now it only calls done if there is an error. Does this change mean the previous behavior was incorrect?

Reply:
Yes - we're done once the `agent.use` called by `withPeerService` is done, not before. IIRC this was causing some 'done called twice' output even in successful tests.